packages/cli/config-management/generators-configuration/src/schemas/RubyGemsOutputLocationSchema.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from "zod";
+
+// It is worth noting that RubyGems API keys need to have the "Push rubygem" permission.
+// Ideally it is also permissioned with index and yank rubygem permissions.
+// Additionally if the creator of the API key has MFA enabled, they must be sure to update their MFA
+// settings to not require MFA for API key usage ("UI and gem signin").
```

Comment:
can we put this in a guide in the docs?

Reply:
not required in this PR

Reply:
https://docs.buildwithfern.com/overview/guides/publishing-to-maven-central example

Reply:
definitely!